src/project-2.js

```diff
@@ -44,62 +79,104 @@ const isPrime = (num) => {
   // hint: a prime number is only evenly divisible by itself and 1
   // hint2: you can solve this using a for loop
   // note: 0 and 1 are NOT considered prime numbers
+  if (num === 0 || num === 1) {
+    return false;
+  }
+  if (num % 2 === 0 || num % 3 === 0) {
+    return false;
+  }
+  return true;
 };
 
 const returnFirst = (arr) => {
   // return the first item from the array
+  return arr[0];
 };
 
 const returnLast = (arr) => {
   // return the last item of the array
+  return arr[arr.length - 1];
 };
 
 const getArrayLength = (arr) => {
   // return the length of the array
+  return arr.length;
 };
 
 const incrementByOne = (arr) => {
   // arr is an array of integers
   // increase each integer by one
   // return the array
+  for (let i = 0; i < arr.length; i++) {
+    arr[i]++;
+  }
+  return arr;
 };
 
 const addItemToArray = (arr, item) => {
   // add the item to the end of the array
   // return the array
+  arr.push(item);
+  return arr;
 };
 
 const addItemToFront = (arr, item) => {
   // add the item to the front of the array
   // return the array
   // hint: use the array method .unshift
+  arr.unshift(item);
+  return arr;
 };
 
 const wordsToSentence = (words) => {
   // words is an array of strings
   // return a string that is all of the words concatenated together
   // spaces need to be between each word
   // example: ['Hello', 'world!'] -> 'Hello world!'
+  let newWords = '';
+  newWords += `${words[0]}`;
+  for (let i = 1; i < words.length; i++) {
+    newWords += ` ${words[i]}`;
+  }
+  return newWords;
 };
 
 const contains = (arr, item) => {
   // check to see if item is inside of arr
   // return true if it is, otherwise return false
+  return arr.includes(item);
```

Comment:
My favorite JS.Array method. Do you know how you'd solve this without it?

Reply:
We could use a for loop and test arr[i] to see if it's equal to item.
We could also use a forEach and sets a flag to true then return that flag's state.
I believe we can also use the for ...in... loop since arr is a valid type and the order is not important. (I read the doc on for...in... where they advise not to use for...in... if the order of index is important)

Reply:
Yup! Great stuff!